src/postamble_minimal.js

```diff
@@ -220,6 +220,16 @@ WebAssembly.instantiate(Module['wasm'], imports).then(function(output) {
   assert(wasmTable);
 #endif
 
+#if AUDIO_WORKLET
+  // If we are in the audio worklet environment, we can only access the Module object
+  // and not the global scope of the main JS script. Therefore we need to export
```

Comment:
Why is this an issue - are audio worklets different types of JS environments?

Reply:
I believe this has to do with how the function [Worklet.addModule()](https://developer.mozilla.org/en-US/docs/Web/API/Worklet/addModule) works:

The AudioWorklet global scope does not have a `importScripts()` function to call like e.g. pthread worker.js does, but the main browser thread must "push" JS script files to be loaded into the worklet scope by calling the function `Worklet.addModule()`.

This `Worklet.addModule()` function has a semantical difference that all JS code present there will be treated as if it is part of a module namespace, and as result none of the global variables in the JS file will be visible to the global scope of the AudioWorklet. So to work around that, all the data is routed via the `Module` object.

Reply:
I see, thanks... I guess they wanted to do something more like ES Modules than just `importScripts` which is like `eval`...